src/features/languages/generated/client/client.json

```diff
@@ -1590,6 +1590,7 @@
         "review.actions.title.registerActionTitle": "Ready to register?",
         "review.birthRegistration.queryError": "An error occurred while fetching birth registration",
         "review.button.approve": "Send for approval",
+        "review.documents.documentForWhom": "{section, select, child {Child's} mother {Mother's} father {Father's} deceased {Deceased's} informant {Informant's} other {Parents'} other {}}",
```

Comment:
This content key was removed in a previous PR. This might have come back in a merge commit. @favour234 Could you please remove this and the french translation as well?